docs/shared_parsers_catalog/system_time.rst

```diff
@@ -1,3 +1,38 @@
-.. automodule:: insights.parsers.system_time
+System time configuration
+=========================
+
+file ``/etc/chrony.conf``
+-------------------------
+.. autoclass:: insights.parsers.system_time.ChronyConf
+   :members:
+   :show-inheritance:
+
+file ``/etc/sysconfig/chronyd``
+-------------------------------
+.. autoclass:: insights.parsers.system_time.ChronydService
+   :members:
+   :show-inheritance:
+
+command ``file -L /etc/localtime``
+----------------------------------
+.. autoclass:: insights.parsers.system_time.LocalTime
+   :members:
+   :show-inheritance:
+
+file ``/etc/ntp.conf``
+----------------------
+.. autoclass:: insights.parsers.system_time.NTP_conf
+   :members:
+   :show-inheritance:
+
+command ``ntptime``
+-------------------
+.. autoclass:: insights.parsers.system_time.NtpTime
+   :members:
+   :show-inheritance:
+
+file ``/etc/sysconfig/ntpd``
+----------------------------
+.. autoclass:: insights.parsers.system_time.NTPDService
```

Comment:
Hmmm - it's not supposed to be.  It's a base class for NTP_conf and ChronyConf, so they both work the same way.  It doesn't have a parser decorator for that reason.  I certainly agree that both NTP_conf and ChronyConf need better documentation if they're going to refer to a class that doesn't exist, but I'd rather not mention the base class so as to prevent rule writers thinking they need to use that class.

How about I update the NTP_conf and ChronyConf documentation with a copy of the documentation in NTPConfParser?  What's the best way of getting good documentation into the parsers that we want people to use?